pages/osx/shortcuts.md

```diff
@@ -0,0 +1,20 @@
+# shortcuts
+
+> Use the Terminal app to run shortcuts from command line instead from the Shortcuts app.
```

Comment:
```suggestion
> Use the Terminal app to run shortcuts from command line instead from the `Shortcuts` app.
```

What about to quote all app names via backticks? I am not sure whether it's a good idea. @navarroaxel